masterlist.yaml

```diff
@@ -3857,6 +3857,30 @@ plugins:
     clean:
       - crc: 0x65AFCE11 # v4.06SE
         util: 'SSEEdit v3.2.2'
+  - name: 'BandolierForNPC.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC
+    tag:
+      - Invent
```

Comment:
You don't need the `Invent` tag as that is for changes to `NPC_` or `CONT` Container records.

The following Bash tags can be added as well to all your plug-ins.
```yaml
    tag:
      - Names
      - Stats
```